### ISSUE TITLE
Imported DOI via Acacia not appearing in recent additions (1.1.1-next)

### ISSUE THREAD

**Issue by rsdoiel** [caltechlibrary/eprinttools#50]
DOI: 10.1021/acsami.1c18243
From George's email: 

```
Completed and pushed out 1 record
 
Yu, Weilai and Young, James L. and Deutsch, Todd G. and Lewis, Nathan S. (2021) Understanding the Stability of Etched or Platinized p-GaInP Photocathodes for Solar-Driven H₂ Evolution. ACS Applied Materials & Interfaces . ISSN 1944-8244. doi:10.1021/acsami.1c18243. (In Press) https://resolver.caltech.edu/CaltechAUTHORS:20201129-123456789
 
Extra verbiage in public display:
 
Available Versions of this Item
Understanding the Stability of Etched or Platinized p-GaInP Photocathodes for Solar-Driven H₂ Evolution. (deposited 29 Nov 2021 08:59) [Currently Displayed]
Commentary/Response Threads
Yu, Weilai and Young, James L. and Deutsch, Todd G. and Lewis, Nathan S. Understanding the Stability of Etched or Platinized p-GaInP Photocathodes for Solar-Driven H₂ Evolution. (deposited 29 Nov 2021 08:59) [Currently Displayed]
 
Does not appear on the Recent Additions page.
```

**Comment by rsdoiel** [comment]
Confirm that I am setting the right timestamp fields for the right publication date values (e.g. types and datestamp).

**Comment by rsdoiel** [comment]
+--------------------+----------+---------------+-----------+------------+------------+----------------+------------+
| id_number          | eprintid | eprint_status | date_type | date       | datestamp  | status_changed | lastmod    |
+--------------------+----------+---------------+-----------+------------+------------+----------------+------------+
| 20211130-123456789 |   112089 | archive       | published | 2021-11-15 | 2021-11-30 | 2021-11-30     | 2021-11-30 |
| 20211130-211842779 |   112096 | archive       | published | 2021-11-DD | 2021-11-30 | 2021-11-30     | 2021-11-30 |
| 20211201-482793884 |   112133 | archive       | inpress   | 2021-11-29 | 2021-12-1  | 2021-12-1      | 2021-12-1  |
+--------------------+----------+---------------+-----------+------------+------------+----------------+------------+

112089 is not listed in the latest editions, I think it should be listed (was imported via Acacia pre 1.1.1-rc2). 
112096 does show up correctly in the latest editions list (was imported with old version of doi2eprintxml)
112133 is not listed in the latest editions, I think it should be listed (was imported via Acacia post 1.1.1-rc2). 

Since it was imported with 1.1.0 version of doi2eprintxml I'll see if I can find a missing field in the new output.

**Comment by rsdoiel** [comment]
Need to confirm if this was fixed in 1.1.1-rc3 or not.

**Comment by rsdoiel** [comment]
Confirmed fix after running `./bin/generate_views --generate list` on the EPrints server.
